app/internal/wm/os_windows.go

```diff
@@ -555,6 +534,49 @@ func (w *window) Option(opts *Options) {
 	}
 }
 
+//SetWindowLayout set size and position of window
+func (w *window) SetWindowLayout(opts *Options) {
+	dpi := windows.GetSystemDPI()
+	cfg := configForDPI(dpi)
+
+	width := int32(cfg.Px(opts.Size.Width))
+	height := int32(cfg.Px(opts.Size.Height))
+
+	// Include the window decorations.
+	wr := windows.Rect{
+		Right:  width,
+		Bottom: height,
+	}
+	dwStyle := uint32(windows.WS_OVERLAPPEDWINDOW)
+	dwExStyle := uint32(windows.WS_EX_APPWINDOW | windows.WS_EX_WINDOWEDGE)
+	windows.AdjustWindowRectEx(&wr, dwStyle, 0, dwExStyle)
+
+	dw, dh := width, height
+	width = wr.Right - wr.Left
+	height = wr.Bottom - wr.Top
+	w.deltas.width = width - dw
+	w.deltas.height = height - dh
+
+	w.opts.Size = opts.Size
+	if opts.Centered {
+		w.CenterWindow(width, height)
+	} else {
+		w.MoveWindowToTopLeft(width, height)
+	}
+}
+
+func (w *window) MoveWindowToTopLeft(width, height int32) {
+	windows.MoveWindow(w.hwnd, 0, 0, width, height, true)
+}
+
+func (w *window) CenterWindow(width, height int32) {
+	screenWidth := int32(windows.GetSystemMetrics(w.hwnd, windows.SM_CXSCREEN))
+	screenHeight := int32(windows.GetSystemMetrics(w.hwnd, windows.SM_CYSCREEN))
+	x := screenWidth/2 - width/2
+	y := screenHeight/2 - height/2
+	windows.MoveWindow(w.hwnd, x, y, width, height, true)
+}
```

Comment:
Inline these functions into SetWindowLayout; they're not worth splitting out. If you do want to split them, at least move the MoveWindow call to SetWindowLayout and separate only the width/height computations.